src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day_thu calendar--month-length_31">
+      <div class="calendar__mockup">
+        <div class="calendar__day calendar__day--background_grey"></div>
+        <div class="calendar__day calendar__day--background_grey"></div>
+        <div class="calendar__day calendar__day--background_grey"></div>
+        <div class="calendar__day calendar__day--background_grey"></div>
+        <div class="calendar__day calendar__day--background_grey"></div>
+        <div class="calendar__day calendar__day--background_grey"></div>
+        <div class="calendar__day calendar__day--background_grey"></div>
+        <div class="calendar__day calendar__day--background_grey"></div>
+        <div class="calendar__day calendar__day--background_grey"></div>
+        <div class="calendar__day calendar__day--background_grey"></div>
+        <div class="calendar__day calendar__day--background_grey"></div>
+        <div class="calendar__day calendar__day--background_grey"></div>
+        <div class="calendar__day calendar__day--background_grey"></div>
+        <div class="calendar__day calendar__day--background_grey"></div>
+        <div class="calendar__day calendar__day--background_grey"></div>
+        <div class="calendar__day calendar__day--background_grey"></div>
+        <div class="calendar__day calendar__day--background_grey"></div>
+        <div class="calendar__day calendar__day--background_grey"></div>
+        <div class="calendar__day calendar__day--background_grey"></div>
+        <div class="calendar__day calendar__day--background_grey"></div>
+        <div class="calendar__day calendar__day--background_grey"></div>
+        <div class="calendar__day calendar__day--background_grey"></div>
+        <div class="calendar__day calendar__day--background_grey"></div>
+        <div class="calendar__day calendar__day--background_grey"></div>
+        <div class="calendar__day calendar__day--background_grey"></div>
+        <div class="calendar__day calendar__day--background_grey"></div>
+        <div class="calendar__day calendar__day--background_grey"></div>
+        <div class="calendar__day calendar__day--background_grey"></div>
+        <div class="calendar__day calendar__day--background_grey"></div>
+        <div class="calendar__day calendar__day--background_grey"></div>
+        <div class="calendar__day calendar__day--background_grey"></div>
```

Comment:
Don't add a modifier for the default styles

Reply:
Well noted ,  back ground color is a default setting , thus to be add directly to the element.
Good suggestion.